src/App.js

```diff
@@ -1,5 +1,21 @@
+import datas from "./datas/datas.js";
+import {Console} from '@woowacourse/mission-utils';
+import RacingCarUtils from "./utils/RacingCarUtils.js";
+import InputHandler from "./utils/InputHandler.js";
+
+
 class App {
-  async run() {}
+    async run() {
+        const playerList = await InputHandler.getInput(datas.INSTRUCTION_CARS, (str) => str.split(','))
+        let carList = RacingCarUtils.createCarListByPlayerList(playerList);
+
+        const tryNumber = await InputHandler.getInput(datas.INSTRUCTION_TRY_NUMBER)
+
+        Console.print("\n실행결과")
+        carList = RacingCarUtils.race(RacingCarUtils.deepCopy(carList), tryNumber);
+        const winners = RacingCarUtils.findWinners(carList).join(', ')
+        Console.print("최종 우승자 : " + winners)
+    }
```

Comment:
많은 곳에서 재할당의 불확실성과 같은 이유들로 let 사용을 피하는 것을 권장하고 있어요
물론 이번 미션과 같이 규모가 작은 경우엔 혼동하지 않겠지만, 미리 습관을 들여 놓는 것도 좋다고 생각합니다!
[reference](https://github.com/airbnb/javascript?tab=readme-ov-file#references--prefer-const)

많은 방법이 있겠지만, 작성해 주신 구조를 유지하면서 적용한다면 아래와 같이 작성해도 좋겠네요!

```suggestion
    async run() {
        const playerList = await InputHandler.getInput(datas.INSTRUCTION_CARS, (str) => str.split(','));
        const initialCarList = RacingCarUtils.createCarListByPlayerList(playerList);

        const tryNumber = await InputHandler.getInput(datas.INSTRUCTION_TRY_NUMBER);

        Console.print("\n실행결과");
        const carListAfterRace = RacingCarUtils.race(RacingCarUtils.deepCopy(initialCarList), tryNumber);
        const winners = RacingCarUtils.findWinners(carListAfterRace).join(', ');
        Console.print("최종 우승자 : " + winners);
    }
```

Reply:
아.. 이 부분은 실수했네요
불변성을 지키고 싶다고 생각해서 deepCopy를 추가해보았는데 let으로 선언해버려서 
이로 인한 이점을 잃어버렸네요 ㅠㅠ
앞으로 더 꼼꼼하게 검수해야겠어요

감사합니다!